src/app/components/create-edit-event-page/create-edit-event-page.component.html

```diff
@@ -0,0 +1,348 @@
+                                        {{ category }}
+                                    </mat-option>
+                                </mat-select>
+                                <mat-error 
+                                    *ngIf="eventGeneralForm.get('category')?.hasError('required')">
+                                    Category is <strong>required</strong>
+                                </mat-error>
+                            </mat-form-field>
+                        </div>
+                    </div>
+
+                    <div class="event-form-item-wrapper" formGroupName="joinEvent">
+                        <div class="event-form-item-container">
+                            <h4>Join Deadline</h4>
+                            <mat-form-field class="event-form-item" appearance="fill">
+                                <mat-label>MM-DD-YYYY</mat-label>
+                                <input matInput [matDatepicker]="deadlineDatePicker" formControlName="joinDeadlineDate">
+                                <mat-datepicker-toggle matSuffix [for]="deadlineDatePicker"></mat-datepicker-toggle>
+                                <mat-datepicker #deadlineDatePicker></mat-datepicker>
+                                <mat-error
+                                    *ngIf="eventGeneralForm.get('joinEvent')?.get('joinDeadlineDate')?.hasError('required')">
+                                    Deadline date for event is <strong>required</strong>
+                                </mat-error>
+                            </mat-form-field>
+                        </div>
+
+                        <div class="event-form-item-container">
+                            <h4>Joind Deadline Time</h4>
+                            <mat-form-field class="event-form-item">
+                                <input matInput [ngxMatTimepicker]="deadlineTimePicker" placeholder="4:00" formControlName="joinDeadlineTime">
+                                <mat-icon matSuffix (click)="deadlineTimePicker.open()">
+                                    watch_later
+                                </mat-icon>
+                                <mat-error 
+                                    *ngIf="eventGeneralForm.get('joinEvent')?.get('joinDeadlineTime')?.hasError('required')">
+                                    Deadline time for event is <strong>required</strong>
+                                </mat-error>
+                            </mat-form-field>
+                            <ngx-mat-timepicker #deadlineTimePicker></ngx-mat-timepicker>
+                        </div>
+
+                    </div>
+                    
+                    <div class="event-form-item-wrapper">
+                        <div class="event-form-item-container">
+                            <mat-checkbox class="checkBox" 
+                                 [(ngModel)]="autocancelChecked" 
```

Comment:
ngModel is for template driven forms, you are using a reactive one. If you need this checkbox to be checked by default, you should set the checked attribute to autocancelChecked